src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DBFilterBase.java

```diff
@@ -237,6 +237,7 @@ public List<BuiltCondition> getAll() {
                   getDocumentIdValue(values.get(0))));
 
         case IN:
+          if (values.isEmpty()) return List.of();
```

Comment:
Why is this supported? and if it's empty array is the whole `$in` condition effectively ignored?

Reply:
In addition, how would this actually reflect the query:

```
{
  key: { $in: [] },
  smth-else: { $eq: 'geroge' }
}
```

I guess this would not work atm? If it would, can you add a test please..

Reply:
No condition is not ignored, basically no document is sent back in response.

Reply:
> I guess this would not work atm? If it would, can you add a test please..

If there is $in with empty array we don't build and run query irrespective of other conditions available as part of filter(everything is `and` operations), this will return empty response.

Reply:
Perfect thanks :+1: